brave/browser-laptop
# Refactor download.js with Aphrodite

including:

- Created `<List>` and `<DownloadList>` in list.js
- Copied itemList.less to commonStyles.js except `&.selected`
- Added aboutPageSectionMargin to global.js

Closes #7255
Addresses #7318

Auditors: @bsclifton @cezaraugusto 

Test Plan:
1. Open about:download
2. Download a file
3. Make sure the styles are not broken


Test Plan:
